docs/tutorials/testing_with_control_center.rst

```diff
@@ -33,6 +33,10 @@ section of our website, and individual links are provided below for convenience:
 
 Hardware Setup
 ==============
+.. warning:: **If you are using a power supply to power your module, it is possible  to damage or destroy your module with aggressive commands.** This is because Vertiq modules can also act as generators. 
+  Power supplies generally cannot absorb the energy generated by the module, while batteries can, which can lead to dangerous voltage spikes when connected to a power supply. 
+  To prevent damage to the module when commanding it aggressively on a power supply, it is recommended to turn on the regeneration voltage protection feature.
```

Comment:
link to the regen voltage protection feature? Not sure if we have one

Reply:
Sadly, we do not have any documentation to link to on it. Hopefully we will in the future